content/docs/ethernet-v0.5/fpga2.md

```diff
@@ -0,0 +1,489 @@
+
+    Created a new partition 1 of type 'Linux' and of size 32 MiB.
+
+    Command (m for help): t
+    Selected partition 1
+    Partition type (type L to list all types): b
+    Changed type of partition 'Linux' to 'W95 FAT32'.
+
+    Command (m for help): n
+    Partition type
+       p   primary (1 primary, 0 extended, 3 free)
+       e   extended (container for logical partitions)
+    Select (default p): p
+    Partition number (2-4, default 2): 
+    First sector (67584-15564799, default 67584): 
+    Last sector, +sectors or +size{K,M,G,T,P} (67584-15564799, default 15564799): +600M
+
+    Created a new partition 2 of type 'Linux' and of size 600 MiB.
+
+    Command (m for help): n
+    Partition type
+       p   primary (2 primary, 0 extended, 2 free)
+       e   extended (container for logical partitions)
+    Select (default p): p
+    Partition number (3,4, default 3): 
+    First sector (1296384-15564799, default 1296384): 
+    Last sector, +sectors or +size{K,M,G,T,P} (1296384-15564799, default 15564799): +1G
+
+    Created a new partition 3 of type 'Linux' and of size 1 GiB.
+
+    Command (m for help): t
+    Partition number (1-3, default 3): 
+    Partition type (type L to list all types): 82
+
+    Changed type of partition 'Linux' to 'Linux swap / Solaris'.
+
+    Command (m for help): p
+    Disk /dev/sdc: 7.4 GiB, 7969177600 bytes, 15564800 sectors
+    Units: sectors of 1 * 512 = 512 bytes
+    Sector size (logical/physical): 512 bytes / 512 bytes
+    I/O size (minimum/optimal): 512 bytes / 512 bytes
+    Disklabel type: dos
+    Disk identifier: 0x074e0e18
+
+    Device     Boot   Start     End Sectors  Size Id Type
+    /dev/sdc1          2048   67583   65536   32M  b W95 FAT32
+    /dev/sdc2         67584 1296383 1228800  600M 83 Linux
+    /dev/sdc3       1296384 3393535 2097152    1G 82 Linux swap / Solaris
+
+    Command (m for help): w
+    The partition table has been altered.
+    Calling ioctl() to re-read partition table.
+    Re-reading the partition table failed.: Device or resource busy
+
+    The kernel still uses the old table. The new table will be used at the next reboot or after you run partprobe(8) or kpartx(8).
+
+To ensure the new partitions are recognised, remove and reinsert the card.
+
+## Setting the partitions (use the image downloaded at the top of this page)
+
+    sudo mkfs.msdos /dev/sdc1
+    sudo dd if=rootfs.ext2 of=/dev/sdc2 bs=64M
+    sudo mkswap /dev/sdc3
+
+## Optional: set up the time-server
+
+    sudo vi /etc/xinetd.d/time
+    #Change the first occurence of "disable=yes" to "disable=no". This enables the time service.
+    sudo /etc/init.d/xinetd restart
+
+## SD booting (kernel via Ethernet)
+
+    cd $TOP/fpga/board/nexys4_ddr
+    make ethersd
+
+## SD booting (standalone)
+
+    cd $TOP/fpga/board/nexys4_ddr
+    cp *.bin /media/jrrk2/26CD-9823/ # or wherever your SD-card partition is mounted
+    make boot cfgmem-updated program-cfgmem-updated
```

Comment:
Should clarify that last line isn't necessary if using the prebuilt bitstream.

From these tutorial instructions it's not very clear which boot configurations are supported if using the prebuilt bitstream from github (all of them? a subset?).